src/test/java/io/airlift/compress/zstd/TestUtil.java

```diff
@@ -42,4 +43,13 @@ public void testGet24BitLittleEndian(byte[] bytes, int offset, int value)
         long inputAddress = ARRAY_BYTE_BASE_OFFSET + offset;
         assertEquals(get24BitLittleEndian(inputBase, inputAddress), value);
     }
+
+    @Test(dataProvider = "test24bitIntegers")
+    public void testPut24BitLittleEndian(byte[] bytes, int offset, int value)
+    {
+        Object outputBase = bytes.clone();
```

Comment:
The output should start as an empty array. Otherwise, a no-op implementation of `put24BitLittleEndian` would result in tests succeeding. Replace this with:

```
Object outputBase = new byte[bytes.length];
```

Reply:
Replaced